### PR TITLE
Upgrade

### PR DESCRIPTION
**Pull request by GuillaumeAmat**
I missed [that commit](https://github.com/ekino/docker-buildbox/commit/7605f7ac557974426eaf4ad18da564cb9edb166f) before starting to upgrade the tools... But anyway, I was able to push it a bit further.

### CODE REVIEW

File: java/6/Dockerfile [hunk 1]
```diff
@@ -1,6 +1,8 @@
 FROM debian:squeeze
 MAINTAINER Jean-Baptiste Hembise <jean-baptiste.hembise@ekino.com>
 
+ARG MAVEN_VERSION
```
[the remark this answers is not in the document]
Comment (GuillaumeAmat) [reply java/6/Dockerfile]: Done

File: node/Dockerfile [hunk 2]
```diff
@@ -8,6 +8,7 @@ ENV DEBIAN_FRONTEND=noninteractive \
 
 ARG NODE_VERSION
 ARG NPM_VERSION
+ARG NVM_VERSION=0.33.11
```
[the remark this answers is not in the document]
Comment (GuillaumeAmat) [reply node/Dockerfile]: Done

**Comment by GuillaumeAmat** [comment]
I'm looking at the CI errors

**Comment by GuillaumeAmat** [comment]
It's all green @mremi 👌

File: CHANGELOG.md [hunk 3]
```diff
@@ -14,19 +14,21 @@ Versions
...
 * Install PEAR in the PHP 5.3 image
-* Update Alpine Version for sonar image: 3.8
+* Update Alpine Version for ansible, aws and sonar image: 3.8
 * Add `build-base`, `libffi-dev` and `openldap-dev` packages in Python images
 * Upgrade Pip version in Python images
 * Update dind-aws to use 18.06.1-ce-dind
-* Fix ansible build (move from python 2.7.14-r0 to 2.7.15-r0)
+* Fix ansible build (move from python 2.7.14-r0 to 2.7.15-r1)
 * Java image : fix aws failed when removing pip
+* Upgrade Maven version : 3.5.4
```
Comment (mremi) [inline CHANGELOG.md]: 3.2.5?

**Pull request merged**